packages/data-explorer-ui/src/components/common/Button/button.styles.ts

```diff
@@ -59,3 +59,13 @@ export const TextButtonPrimary = styled(MButtonBase)`
     text-decoration: underline;
   }
 `;
+
+// White text button
+export const TextButtonWhite = styled(MButtonBase)`
+  color: ${({ theme }) => theme.palette.common.white};
```

Comment:
insert above line `65` `${textBodySmall400};` and import from `import { textBodySmall400 } from "../../../styles/common/mixins/fonts";` these font styles will come in handy!

Reply:
The `TextButtonPrimary` above was written prior to the availability of the fonts described above - we should try to use them as much as possible going forwards!